projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-row-dimension-mrl-row.component.ts

```diff
@@ -114,4 +115,13 @@ export class IgxPivotRowDimensionMrlRowComponent extends IgxGridHeaderRowCompone
         }
         return resWidth;
     }
+
+    /**
+     * @hidden @internal
+     */
+    public getGroupKey(group: IPivotGridHorizontalGroup) {
```

Comment:
Use `protected` for template-specific methods/props like that :) Angular has allowed that for a while now :)
```suggestion
    protected getGroupKey(group: IPivotGridHorizontalGroup) {
```